modules/claims_api/app/workers/claims_api/report_unsuccessful_submissions.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module ClaimsApi
+  class ReportUnsuccessfulSubmissions
+    include Sidekiq::Worker
+
+    def perform
+      if Settings.claims_api.unsuccessful_report_enabled
+        @to = Time.zone.now
+        @from = @to.monday? ? 7.days.ago : 1.day.ago
+        @consumers = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to).pluck(:source).uniq
```

Comment:
I might be missing something but it looks like @consumers is only used within the total method? Is there a reason its defined up in this scope?

Reply:
I think this was more of the pattern used in the original email and I just copied it over for use here to be consistent, worth a refactor in the future for sure.